components/ws-manager/pkg/manager/monitor.go

```diff
@@ -201,9 +202,11 @@ func (m *Monitor) onVolumesnapshotEvent(evt watch.Event) error {
 		m.eventRecorder.Eventf(&pod, corev1.EventTypeNormal, "VolumeSnapshot", "Volume snapshot %q is ready to use", vs.Name)
 	}
 
+	m.notifyPodMapLock.Lock()
 	if m.notifyPod[podName] == nil {
 		m.notifyPod[podName] = make(chan string)
 	}
+	m.notifyPodMapLock.Unlock()
 	m.notifyPod[podName] <- vsc
```

Comment:
I thought the race condition happened at this line.
If yes, the `m.notifyPodMapLock.Unlock()` should be at line 211.
Am I wrong?